.github/workflows/test.yml

```diff
@@ -41,25 +41,15 @@ jobs:
         with:
```

Comment:
I think the `setup-python` action is not needed anymore if we use `setup-uv` (see [here](https://github.com/astral-sh/setup-uv?tab=readme-ov-file#do-i-still-need-actionssetup-python-alongside-setup-uv)).